spec/support/pundit.rb

```diff
@@ -0,0 +1,17 @@
+module PunditViewPolicy
+  extend ActiveSupport::Concern
+
+  included do
+    before do
+      controller.singleton_class.class_eval do
+        def policy(instance)
+          Class.new do
+            def method_missing(*args, &block); true; end
```

Comment:
😩 I'm assuming this is to mock out every pundit policy during tests to always return true?

Do we have to undo this if we want to write specs against the view and still have pundit do its thing?